client/jetpack-cloud/sections/partner-portal/license-product-card/index.tsx

```diff
@@ -41,17 +43,20 @@ export default function LicenseProductCard( props: Props ) {
 
 	useEffect( () => {
 		if ( suggestedProduct ) {
-			if ( product.slug === suggestedProduct ) {
+			// Transform the comma-separated list of products to array.
+			const suggestedProducts = suggestedProduct.split( ',' );
+
+			if ( suggestedProducts.includes( product.slug ) ) {
 				onSelect();
 			}
 		}
-	}, [ onSelect, product, suggestedProduct ] );
+	}, [] );
```

Comment:
Just for clarity: We don't want to specify any dependencies here because we want this to be executed once, right? I don't remember if I made that change specifically.

Reply:
Yes, correct.